pkgs/applications/audio/bitwig-studio/bitwig-studio3.nix

```diff
@@ -1,17 +1,16 @@
-{ fetchurl, bitwig-studio1, pulseaudio, xorg }:
+{ stdenv, fetchurl, bitwig-studio2, xorg, ... }:
 
-bitwig-studio1.overrideAttrs (oldAttrs: rec {
+bitwig-studio2.overrideAttrs (oldAttrs: rec {
```

Comment:
Usually, when packaging multiple versions, we factor out into something like a `common.nix` and then have the specific versions pass in the src and version themselves, rather than chaining them on top of each other with overrides. Here, if we change something idiosyncratic to the base package, we'd rebuild them all.

Reply:
Makes sense. Will have a look at it, but the old versions won't be updated anymore. At least that is what the business model of bitwig says.